recipes/openms-meta/meta.yaml

```diff
@@ -1,24 +1,22 @@
 {% set name = "OpenMS" %}
-{% set version = "2.6.0" %}
+{% set version = "2.7.0" %}
 package:
   name: {{ name|lower  }}-meta
   version: {{ version  }}
 
 source:
   url: https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/{{ version }}/OpenMS-{{ version }}-src.tar.gz
-  sha256: 55f30d09cba420009587e50d70812a4775d012151df8c600c7c39e59079bc10b
+  sha256: 62bce6215bc88ca7d1f1b7abf15cea4061bdcfa10dbb1e652b1d8c2ff9b3f081
   patches:
     - apple_pyoms_nofixup.patch
     - init.py.patch
     - MANIFEST.in.patch
+    - ParamValue_h.patch
 
 build:
   number: 1
-
-# Try this when building with GUI
-#extra:
-#  container:
-#    extended-base: True
+  # Problems with memory usage on gcc+circleci. Needs to be built manually and pushed. And to debug macos
+  skip: True  # [linux]
```

Comment:
@jpfeuffer we are not using CircelCI anymore. Do you dare to test this again here?

Reply:
Yep it worked. Just needed to fix macOS now and did not want to wait so long all the time ;)